src/line.rs

```diff
@@ -0,0 +1,78 @@
+/// A macro for creating a [`Line`] using vec! syntax.
+///
+/// `line!` is similar to the [`vec!`] macro, but it returns a [`Line`] instead of a `Vec`.
+///
+/// # Examples
+///
+/// * Create a [`Line`] containing a vector of [`Span`]s:
+///
+/// ```rust
+/// # use ratatui::prelude::*;
+/// use ratatui_macros::{line, raw};
+///
+/// let line = line!["hello", "world"];
+/// let line = line!["hello".red(), "world".red().bold()];
+/// ```
+///
+/// * Create a [`Line`] from a given [`Span`] and size:
```

Comment:
nit: size makes sense for the same comment from `vec!`, but is ambiguous for `line!`
```suggestion
/// * Create a [`Line`] from a given [`Span`] repeated some amount of times:
```